src/pages/SignUpConfirmation/SignUpConfirmation.tsx

```diff
@@ -1,87 +1,111 @@
+          className={classes.logo}
         >
-          {
-            ownerInfo.logo
-              ? (
-                <img
-                  className={classes.imageLogo}
-                  src={ownerInfo.logo}
-                />
-              )
-              : (
-                <AmpStoriesRoundedIcon
-                  className={classes.iconLogo}
-                />
-              )
-          }
-
-          <h3 className={classes.portalName}>
+          <Logo
+            icon={navigation.title.iconFallbackName}
+            src={ownerInfo.logo}
+          />
+          <Typography variant="h3">
             {portalName}
-          </h3>
-        </div>
+          </Typography>
+        </Grid>
 
-        <div
-          className={classes.closeButtonContainer}
+        <Box
+          display="flex"
+          justifyContent="flex-end"
+          alignItems="center"
+          color={palette.common.white}
           onClick={() => history.push("/")}
+          className={classes.close}
         >
-          <p>
-            {t("signInOrUpView.closeButtonLabel")}
-          </p>
-
-          <CloseRoundedIcon />
-        </div>
-      </header>
-
-      <section className={classes.pageContentContainer}>
-        <div className={classes.signUpCompleteSideContentContainer}>
-          <h1 className={classes.signUpCompleteSideTitle}>
-            {t("signUpConfirmation.titleText")}
-            {name}!
-          </h1>
+          <Box mr={1} clone>
+            <Typography variant="body2" color="inherit">
+              {t("signInOrUpView.closeButtonLabel")}
+            </Typography>
+          </Box>
 
-          <p className={classes.signUpCompleteSideSubtitle}>
-            <>{t("signUpConfirmation.subtitleTextPartOne")}</>
-            <span className={classes.signUpCompleteSideSubtitleBoldPart}>
-              {t("signUpConfirmation.subtitleTextPartTwo")}
-            </span>
-            <>{t("signUpConfirmation.subtitleTextPartThree")}</>
-          </p>
+          <Icon>close</Icon>
+        </Box>
+      </Box>
 
-          <div className={classes.infoBox}>
-            <InfoRoundedIcon className={classes.infoBoxIcon} />
+      <Grid
+        alignSelf="center"
+        component={Box}
+        container
+        direction="column"
+        item
+        py={20}
+        px={6}
+        maxWidth={breakpoints.values.sm}
+        margin="auto"
+      >
+        <Grid item md>
+          <Box pb={3}>
+            <Typography variant="h1">
+              {t("signUpConfirmation.titleText")}
+              {name}!
+            </Typography>
+          </Box>
 
-            <div>
-              <p className={classes.infoBoxText}>
-                {t("signUpConfirmation.infoBoxText")}
-              </p>
-            </div>
-          </div>
-        </div>
+          <Box pb={5}>
+            <Typography variant="body1" color="textSecondary">
+              <>{t("signUpConfirmation.subtitleTextPartOne")}</>
```

Comment:
why do you need the fragment for? see other occurrences below.
```suggestion
              {t("signUpConfirmation.subtitleTextPartOne")}
```

Reply:
Legacy code. Will be removed.